src/StepFunctionGraph/util/JSONBuilder.util.ts

```diff
@@ -0,0 +1,319 @@
+import * as R from 'ramda';
```

Comment:
nit: I would prefer to keep a uniformed naming convention, and place this file in a `utils` directory rather than `util`

Reply:
@amirkibbar you have a strong preference either way? We actually have several `util` folders and only one other `utils` folder. I don't mind either way, but would be more work to change them all to `utils`.

Reply:
I'm ok with whatever there's more of. So if `util` is used more - this is fine.

Reply:
👍🏾  done.